avocado/utils/software_manager/backends/rpm.py

```diff
@@ -34,7 +34,7 @@ def _check_installed_version(self, name, version):
         :param version: Package version.
         """
         cmd = (self.lowlevel_base_cmd + ' -q --qf %{VERSION} ' + name)
-        inst_version = process.system_output(cmd, ignore_status=True)
+        inst_version = process.system_output(cmd, ignore_status=True).decode()
```

Comment:
The best way, IMO, to achieve this is:

```python
inst_version = process.run(cmd, ignore_status=True).stdout_text
```

Because:

1. It discards STDERR
2. It makes it clear that `inst_version` is a string

Reply:
Thank u for the suggestion. I'll make the change.